net-gateway/src/endpoints/files/pcap_files.rs

```diff
@@ -14,20 +12,19 @@ use net_core_api::core::decoder_api::Decoder;
 use net_core_api::core::typed_api::Typed;
 use net_core_api::core::encoder_api::Encoder;
 
-use net_inserter_api::api::network_packet::network_packet::NetworkPacketDTO;
 use net_inserter_api::api::pcap_file::InsertPcapFileDTO;
 use net_token_verifier::fusion_auth::fusion_auth_verifier::FusionAuthVerifier;
 
 use crate::core::quinn_client_endpoint_manager::QuinnClientEndpointManager;
-use crate::endpoints::files::core::network_packet::NetworkPacket;
+use crate::core::user_facing_error::UserFacingError;
 use crate::{authorization, config::Config};
 
 #[post("/pcap-files")]
 async fn pcap_files(
     config: web::Data<Config>,
     req: HttpRequest,
     mut payload: Multipart
-) -> impl Responder {
+) -> Result<&'static str, UserFacingError> {
```

Comment:
Are we really need to return `Result::Ok<&'static str>`? I don`t get it

Reply:
At least return something. It forms HttpResponse::Ok().body(...) in the background